core-web/apps/dotcms-ui/src/app/portlets/dot-containers/dot-container-create/dot-container-properties/dot-container-properties.component.spec.ts

```diff
@@ -346,5 +346,15 @@ describe('DotContainerPropertiesComponent', () => {
             fixture.detectChanges();
             expect(de.query(By.css('[data-testId="saveBtn"]')).attributes.disabled).toBeDefined();
         });
+
+        it('should save button disable after save', fakeAsync(() => {
+            comp.form.get('title').setValue('Hello');
+            fixture.detectChanges();
+            const saveBtn = de.query(By.css('[data-testId="saveBtn"]'));
+            saveBtn.triggerEventHandler('click');
+            tick(200);
```

Comment:
is this necessary ?

Reply:
Yes because we need to make sure after the save button is disabled.